commands.go

```diff
@@ -1677,16 +1677,22 @@ func (c *commandable) GeoAdd(key string, geoLocation ...*GeoLocation) *IntCmd {
 }
 
 func (c *commandable) GeoRadius(query *GeoRadiusQuery) *GeoLocationCmd {
```

Comment:
I think signature here should be:

```
func (c *commandable) GeoRadius(key, longitude, latitude, query *GeoRadiusQuery) *GeoLocationCmd
func (c *commandable) GeoRadiusByMember(key, member, query *GeoRadiusQuery) *GeoLocationCmd
```

and underneath you can use some general function. Feel free to leave this change for me.